manifests/module/director.pp

```diff
@@ -136,6 +137,22 @@
     $kickstart_settings = {}
   }
 
+  if $service {
+    user { 'icingadirector':
+      home       => '/var/lib/icingadirector',
+      groups     => ['icingaweb2'],
+      managehome => true,
+      shell      => '/bin/false',
+    }
+    -> systemd::unit_file { 'icinga-director.service':
+      source => "puppet:///modules/${module_name}/director/icinga-director.service",
```

Comment:
If using the source directive your service file belongs into files/ not into templates. Otherwise it can't be found:

`Error: /Stage[main]/Icingaweb2::Module::Director/Systemd::Unit_file[icinga-director.service]/File[/etc/systemd/system/icinga-director.service]: Could not evaluate: Could not retrieve information from environment development source(s) puppet:///modules/icingaweb2/director/icinga-director.service
`

Reply:
implemented in 158c975